chain/processor.go

```diff
@@ -55,53 +51,14 @@ func (b *StatelessBlock) Execute(
 
 		stateKeys, err := tx.StateKeys(sm)
 		if err != nil {
+			f.Stop()
 			e.Stop()
 			return nil, nil, err
 		}
+		// Fetch keys from disk or check if it's in cache
+		wg := f.Lookup(ctx, tx.ID(), stateKeys)
```

Comment:
We should note here that we fetch all keys from disk (whether or not they are read/write/allocate perms because we use the existing value on-disk for fee calculation)

Reply:
https://github.com/ava-labs/hypersdk/pull/751#discussion_r1513480075

If we require this, then we don't need to leave this comment.